src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +425,180 @@ public void SetFocus(bool focused)
+            }
+        }
+
+        /// <summary>
+        /// Get cookies for URL.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <returns>'cookie string' on success, otherwise 'null'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string GetCookies (string url)
+        {
+            return Interop.ChromiumEwk.ewk_view_get_cookies_for_url(_realHandle, url);
+        }
+
+        /// <summary>
+        /// Exit full screen.
+        /// </summary>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool ExitFullscreen ()
+        {
+            return Interop.ChromiumEwk.ewk_view_fullscreen_exit(_realHandle);
+        }
+
+        /// <summary>
+        /// Gets the current load progress of the page.
+        /// </summary>
+        /// <returns>'value 0.0 to 1.0' on success, otherwise '-1.0'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public double LoadProgres
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_view_load_progress_get(_realHandle);
+            }
+        }
+
+        /// <summary>
+        /// Sends the orientation of the device.
+        /// </summary>
+        /// <param name="orientation">The new orientation of the device in degree.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void SendOrientation (Orientation orientation)
+        {
+            Interop.ChromiumEwk.ewk_view_orientation_send(_realHandle, orientation);
+        }
+
+        /// <summary>
+        /// Suspends the operation associated with the view.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Suspend ()
+        {
+            Interop.ChromiumEwk.ewk_view_suspend(_realHandle);
+        }
+
+        /// <summary>
+        /// Resumes the operation associated with the view.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Resume ()
+        {
+            Interop.ChromiumEwk.ewk_view_resume(_realHandle);
+        }
+
+        /// <summary>
+        /// Gets the current scale factor of the page.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public double Scale
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_view_scale_get(_realHandle);
+            }
+        }
+
+        /// <summary>
+        /// Sets the current scale factor of the page.
+        /// </summary>
+        /// <param name="scaleFactor,">A new level to set.</param>
+        /// <param name="scrollTo">The class Point object with X, Y coordinates.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetViewScale (double scaleFactor, Point scrollTo)
```

Comment:
It should be void, in C# we don't return boolean on success.

Reply:
Done